src/pages/resources/reference/push-notifications/templates/tabs/catalog.md

```diff
@@ -0,0 +1,105 @@
+| Badge Count | ⛔️ | `adb_n_count` | string | Value to be used on app's badge. |
+| Priority | ⛔️ | `adb_n_priority` | string | Designates the notification priority for Android - [importance](https://developer.android.com/reference/android/app/NotificationChannel#setImportance(int)) for API >= 26 (Android 8.0) or [priority](https://developer.android.com/reference/androidx/core/app/NotificationCompat.Builder#setPriority(int)) for API < 26 |
+| Tag | ⛔️ | `adb_tag` | string | Identifier used to replace existing notifications in the notification drawer. If not specified, each request creates a new notification. If specified and a notification with the same tag is already being shown, the new notification replaces the existing one in the notification drawer. |
+| Sticky | ⛔️ | `adb_sticky` | string | Boolean value represented as a string "true" or "false".<br />When set to false or unset, the notification is automatically dismissed when the user clicks it in the panel. When set to true, the notification persists even when the user clicks it. |
+| Ticker | ⛔️ | `adb_ticker` | string | Sets the "ticker" text, which is sent to accessibility services. Prior to API level 21 (Lollipop), sets the text that is displayed in the status bar when the notification first arrives. |
+| Color - Small Icon | ⛔️ | `adb_clr_icon` | string | Color for the notification's small icon.<br />Represented as six character hex, e.g. `00FF00` |
+| Color - Background | ⛔️ | `adb_clr_bg` | string | Color for notification's background.<br />Represented as six character hex, e.g. `00FF00` |
+| Call to action button text | ✅ | `adb_cta_txt` | string | Text to be shown on the CTA button |
+| Call to action button link | ✅ | `adb_cta_uri` | string | URI to be handled when the user clicks the CTA button |
+| Call to action button color | ✅ | `adb_cta_clr` | string | Color for the CTA button. Represented as six character hex, e.g. `00FF00` |
+| Call to action button text color | ✅ | `adb_cta_txt_clr` | string | Color for the CTA button text. Represented as six character hex, e.g. `00FF00` |
+| Display type | ✅ | `adb_display` | string | Determines if the layout of the catalog goes left-to-right or top-to-bottom. |
+| Catalog Items | ✅ | `adb_items` | string | Three entries describing the items in the product catalog.<br />**The value is an encoded JSON string.**<br />When decoded, the string contains an array of the following objects:<br /><ul><li>`title` (required) - Text to use in the title if this product is selected</li><li>`body`(required) - Text to use in the body if this product is selected</li><li>`img` (required) - URI to an image to use in notification when this product is selected.</li><li>`price` (required) - Price of this product to display when the notification is selected.</li><li>`uri` (required) - URI to be handled when the user clicks the large image of the selected item.</li></ul> |
```

Comment:
can you add a space between `body` and `(required)` to be consistent with the other list items?